core_chat_data/src/commonMain/kotlin/net/thechance/mena/core_chat/data/di/DataSourceModule.kt

```diff
@@ -0,0 +1,5 @@
+package net.thechance.mena.core_chat.data.di
+
+import org.koin.core.module.Module
+
+internal expect val dataSourceModule: Module
```

Comment:
```suggestion
internal val datasourceModule: Module
```
what about making the datasourceModule not actual but the creation of the device provider itself as a expect function with koin scope so u can use `get()` inside androidMaint target

Reply:
finally done